test/unit/trainers/uri_trainer_test.rb

```diff
@@ -38,6 +38,15 @@ def test_train
           end
         end
       end
+
+      def test_train_on_trained_domain
```

Comment:
I would rather this be descriptive of what behaviour it's testing for.

e.g. `test_train_on_trained_domain_returns_without_stepping_through_presets`